Day3/README.md

```diff
@@ -106,6 +106,11 @@ public class HammingDistance {
  */
  #include<iostream>
  #include<string>
+<<<<<<< HEAD
```

Comment:
Remove all the  `<<<<<<<< HEAD`  and extra lines that were added to help resolve merge conflicts.
No need for them in the code